vdist/configuration.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+
+import string
+import sys
+# The ConfigParser module has been renamed to configparser in Python 3
+if sys.version_info[0] == 3:
+    import configparser
+else:
+    import ConfigParser as configparser
+
+import vdist.source as source
+
+LISTABLE_ARGUMENTS = {"source_git", "source_git_directory", "runtime_deps",
```

Comment:
They are not only the keys in the configuration files, they can be too arguments passed directly through launcher as console arguments. Both keys from textual configuration files and arguments from console launcher have to be processed to create a proper `Configuration` instance. I used "arguments" to remark that those parameters are given by user, but we can use what you prefer.